src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.cloud.config.import-check.enabled=false
+server.port: 8081
+spring.datasource.driver: com.mysql.jdbc.Driver
+spring.datasource.url: jdbc:mysql://localhost:3306/user
+spring.datasource.username: root
+spring.datasource.password: root
```

Comment:
minor: even if it a dummy application. make practice of not commiting sensitive information like commiting passwords